Scripts/Tracking/Follow/Modifier/ModifyRigidbodyVelocity.cs

```diff
@@ -0,0 +1,106 @@
+﻿namespace VRTK.Core.Tracking.Follow.Modifier
+{
+    using UnityEngine;
+    using VRTK.Core.Extension;
+
+    /// <summary>
+    /// Updates the position and rotation of the source <see cref="Transform"/> by applying velocity and angular velocity to it's <see cref="Rigidbody"/> to track the target <see cref="Transform"/> position and rotation.

```

Comment:
Typo:
- `it's` ➡️ `its`
- `<see cref="Transform"/>` ➡️ `<see cref="Transform"/>'s`